src/main/java/com/minecolonies/entity/ai/util/ItemStorage.java

```diff
@@ -0,0 +1,102 @@
+package com.minecolonies.entity.ai.util;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Used to store an item with various informations to compare items later on.
+ */
+public class ItemStorage
+{
+    /**
+     * The item to store.
+     */
+    private final Item item;
+
+    /**
+     * The damage value.
+     */
+    private final int damageValue;
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)